RevenueCatUI/CustomerCenter/ViewModels/SubscriptionDetailViewModel.swift

```diff
@@ -37,6 +37,30 @@ final class SubscriptionDetailViewModel: BaseManageSubscriptionViewModel {
         purchaseInformation?.store != .appStore
     }
 
+    var hasActiveSubscription: Bool {
+        !customerInfoViewModel.subscriptionsSection.isEmpty
+    }
```

Comment:
Tickets work only for subscriptions then? We won't allow it for virtual currencies, and non-subscriptions. Is that correct?